.github/workflows/codecov.yaml

```
@@ -14,26 +14,11 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      # This step ensures that the "LITERAL" value is given to the actions.
-      # Using `jq` sends it uninterepted to the action which is sort of "hackish" and doesn't ensure the separation of concerns.
-      - name: Get SUI and NodeJS Versions
-        shell: bash
-        run: |
-          SUI_VERSION=$(jq -r '.SUI_VERSION' "version.json")
-          echo "SUI_VERSION=$SUI_VERSION" >> $GITHUB_ENV
-
-          NODEJS_VERSION=$(jq -r '.engines.node' "package.json")
-          echo "NODEJS_VERSION=$NODEJS_VERSION" >> $GITHUB_ENV
-
-      - name: Setup Sui CLI and install dependencies
-        uses: ./.github/actions/setup-sui
-        with:
-          sui-version: ${{ env.SUI_VERSION }}
-
-      - name: Setup node.js and install dependencies
-        uses: ./.github/actions/nodejs-ci
+      - name: Setup the required tools (Sui and nodejs)
```

```suggestion
      - name: Setup tools
```
no need to specify everywhere, it can be seen from logs